packages/web/config/chain-infos.ts

```diff
@@ -136,6 +136,46 @@ const chainInfos = (
           isStakeCurrency: true,
           isFeeCurrency: true,
         },
+        {
+          type: "cw20",
+          contractAddress: "secret12rcvz0umvk875kd6a803txhtlu7y0pnd73kcej",
+          coinDenom: "ALTER",
+          coinMinimalDenom:
+            "cw20:secret12rcvz0umvk875kd6a803txhtlu7y0pnd73kcej:ALTER",
+          coinDecimals: 6,
+          coinGeckoId: "alter",
+          coinImageUrl: "/tokens/alter.jpg",
```

Comment:
do we have a png or svg(preferred) with transparent corners?

Reply:
Yes, I'll update it

Reply:
FYI, The chain registry appears to now have nice images for each of these tokens' logos.